src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -423,6 +425,14 @@ public IEnumerable<WorkspaceModel> Workspaces
         /// </summary>
         public AuthenticationManager AuthenticationManager { get; set; }
 
+        internal static string DefaultPythonEngine
+        {
+            get
+            {
+                return defaultPythonEngine;
```

Comment:
just curious, why is this not using an auto property?

Reply:
Can use with private set. Will change